aqua/qt5/Portfile

```diff
@@ -666,7 +666,7 @@ array set modules {
     }
     qtwebengine {
         {}
-        "port:python27 port:py27-ply path:bin/ninja:ninja port:gperf port:bison port:flex bin:node:nodejs16"
+        "port:python27 port:py27-ply path:bin/ninja:ninja port:gperf port:bison port:flex"
```

Comment:
nodejs dependency moved to special case section since comments are not allowed in the list here.